### PR TITLE
routing added, index page created

### PR DESCRIPTION
**Pull request by edarrochdavies**

### CODE REVIEW

**Comment by edarrochdavies** [comment]
@wdrury-uk   can you review?

File: app.js [hunk 1]
```diff
@@ -1,5 +1,14 @@
 const express = require('express');
 const app = express();
+
+// set the view engine to ejs
```
Comment (wdrury-uk) [inline app.js:4]: Good use of EJS 👍

**Review by wdrury-uk — APPROVED** [review]
All good,

**Comment by wdrury-uk** [comment]
All approvers happy with code changes.

**Pull request merged**